.github/workflows/ci.yml

```diff
@@ -16,10 +16,12 @@ jobs:
 
     strategy:
       matrix:
-        # FIXME add ubuntu-latest, remove macos-latest: https://jira.percona.com/browse/PMM-6166
-        os: [macos-latest]
+        os: [ubuntu-latest]
         go-version: [1.14.x]
-        kubernetes_version: [1.16.8]  # TODO add other versions
+        kubernetes_version:
+          # https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html
+          - 1.15.11
```

Comment:
Why did you add older version?

Reply:
That's what our operator supports on EKS: https://www.percona.com/doc/kubernetes-operator-for-pxc/System-Requirements.html